cmd/rpcdaemon/commands/parity_api.go

```diff
@@ -70,3 +80,11 @@ func (api *ParityAPIImpl) ListStorageKeys(ctx context.Context, account common.Ad
 	}
 	return keys, nil
 }
+
+func (api *ParityAPIImpl) checkBlockNumber(blockNumber rpc.BlockNumberOrHash) error {
+	hash, isHash := blockNumber.Hash()
+	if !isHash || !bytes.Equal(hash.Bytes(), latestTag.Bytes()) {
```

Comment:
You can detect "latest" by doing this (I think `rpc.LatestBlockNumber` is equal to -1 or something like that):
```
blockNumer.Number == rpc.LatestBlockNumber
```

Parsing of the "latest", "pending", and "oldest" happens higher up in the RPC "framework"

Reply:
done